app/components/ui/Cards/FormattedDate.tsx

```diff
@@ -22,10 +17,11 @@ export const FormattedDate = ({
   const isSingleDayEvent = formattedStartDate === formattedEndDate;
 
   if (isSingleDayEvent) {
-    const formattedStartTime = formatEventTime(calendarEvent.startdate);
-    const endTimeDisplay = calendarEvent.endtime
-      ? `- ${formatEventTime(calendarEvent.endtime)}`
-      : "";
+    const startTimeDateStr = `${calendarEvent.startdate} ${calendarEvent.starttime}`;
+    const endTimeDateStr = `${calendarEvent.startdate} ${calendarEvent.endtime}`;
+    const formattedStartTime = formatEventTime(startTimeDateStr);
+    const formattedEndTime = formatEventTime(endTimeDateStr);
+    const endTimeDisplay = calendarEvent.endtime ? `- ${formattedEndTime}` : "";
 
```

Comment:
Can we protect against an invalid start time?

```suggestion
    const timeDisplay = formattedStartTime ? `· ${formattedStartTime} ${endTimeDisplay}` : "";

    return <p>{`${formattedStartDate} ${timeDisplay}`}</p>;
```

Reply:
^ you'll have to delete the old return statement. not sure why it doesn't show that